src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -332,6 +334,42 @@ describe('AnnotationOmega', () => {
   });
 
   describe('annotation actions', () => {
+    describe('replying to an annotation', () => {
+      // nb: There's no reason this logic needs to stay within `AnnotationOmega`
+      // once we've migrated to it; it could happily move to `AnnotationActionBar`
+      it('should show a flash alert if user not logged in', () => {
+        // No logged-in user...
+        fakeStore.profile.returns({});
+
+        const wrapper = createComponent();
+
+        wrapper
+          .find('AnnotationActionBar')
+          .props()
+          .onReply();
+
+        assert.calledOnce(fakeFlash.error);
+        assert.notCalled(fakeAnnotationsService.reply);
+      });
+
+      it('should proxy to the annotationsService', () => {
```

Comment:
I know that the Python code contains test descriptions like this, but I find it more helpful to phrase test descriptions in terms of the effect for users ("it should create a reply") if possible rather than something abstract ("should call the X.Y method"). My reasoning is that it makes it easier to determine what is likely to have broken when I see the test failure.